src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -71,5 +71,17 @@ public function process(ContainerBuilder $container)
         if ($container->has('assets.packages')) {
             $container->getDefinition('twig.extension.assets')->addTag('twig.extension');
         }
+
+        if (class_exists('Symfony\Component\Yaml\Parser')) {
```

Comment:
That's for another PR as this is not the only place where we are doing this in a compiler pass.